src/gui/qgslegendpatchshapebutton.cpp

```diff
@@ -187,6 +188,35 @@ void QgsLegendPatchShapeButton::prepareMenu()
   mMenu->addAction( defaultAction );
   connect( defaultAction, &QAction::triggered, this, [ = ] { setToDefault(); emit changed(); } );
 
+  mMenu->addSeparator();
+
+  QStringList patchNames = QgsStyle::defaultStyle()->symbolsOfFavorite( QgsStyle::LegendPatchShapeEntity );
```

Comment:
Nice use of favorites. I've been meaning to do that for our symbol buttons too.

Reply:
I copied it from your color ramp button work :stuck_out_tongue:

Reply:
:bow: